ext/fetch/22_body.js

```diff
@@ -191,6 +199,7 @@ class InnerBody {
     const { 0: out1, 1: out2 } = this.stream.tee();
     this.streamOrStatic = out1;
     const second = new InnerBody(out2);
+    // deno-lint-ignore prefer-primordials
     second.source = core.deserialize(core.serialize(this.source));
```

Comment:
Since there are many false positives for the `source` property, I will not check it with deno_lint.